### PR TITLE
Adds a filtering- and sorting-service for the member-list

### LINKED ISSUE #368: Sorting & filtering in MemberList
- [ ] filter by gender
- [ ] filter by meeting/committee

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
- Also includes filtering by committees/meetings
- Also makes the OsFilter interface generic

Fixes #368

### CODE REVIEW

File: client/src/app/management/services/member-filter.service.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                ]
+            },
+            {
+                property: 'gender',
+                label: this.translate.instant('Gender'),
+                options: [
+                    ...genders.map(gender => ({ condition: gender, label: this.translate.instant(gender) })),
+                    { condition: '', label: this.translate.instant('No gender') }
+                ]
+            },
```
Comment (tsiegleauq) [inline client/src/app/management/services/member-filter.service.ts:78]: Remove.
This cannot be filtered but only sorted. The server currently accepts any string for gender, for which filtering will not work

**Review by tsiegleauq — CHANGES_REQUESTED** [review]

File: client/src/app/management/services/member-filter.service.ts (hunk 1 above)
Comment (GabrielInTheWorld) [reply client/src/app/management/services/member-filter.service.ts:78]: Then we should adjust the server to accept only an enum of genders to achieve this "feature". @emanuelschuetze mentioned, that we should filter by gender.

Comment (GabrielInTheWorld) [reply client/src/app/management/services/member-filter.service.ts]: For the first time, I have removed that lines. When OpenSlides/openslides-backend#847 is resolved these filters can be adjusted accordingly.

Comment (tsiegleauq) [reply client/src/app/management/services/member-filter.service.ts]: commenting them out and putting the issue in the comment would have been enough

**Review by tsiegleauq — APPROVED** [review]

**Pull request merged**